vue/sbc-common-components/src/components/SbcSystemBanner.vue

```diff
@@ -1,43 +1,35 @@
 <template>
   <v-alert
-    v-model="show"
+    v-model="state.show"
```

Comment:
Not sure if there is a better way of doing this with script setup

Reply:
see my comment below -- should make it a lot nicer